drake/systems/CMakeLists.txt

```diff
@@ -8,12 +8,20 @@ include_directories(${PROJECT_SOURCE_DIR}/util)
 include_directories(${PROJECT_SOURCE_DIR}/systems)
 include_directories(${PROJECT_SOURCE_DIR}/systems/frames)
 
+if (LCM_FOUND AND NOT WIN32)  # win32 won't be hard... just have to add declspecs
```

Comment:
...and make sure that there are no internal compiler errors. I wouldn't postpone building this on windows too long; best to fix the errors while it's all still fresh in memory.